<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreQuaternionEditorWidget</class>
 <widget class="QWidget" name="CoreQuaternionEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>111</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="quaternionXLabel">
     <property name="text">
      <string>x:</string>
     </property>
     <property name="buddy">
      <cstring>quternionXDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="quternionXDoubleSpinBox">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="quaternionYLabel">
     <property name="text">
      <string>y:</string>
     </property>
     <property name="buddy">
      <cstring>quternionYDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="quternionYDoubleSpinBox">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="quaternionZLabel">
     <property name="text">
      <string>z:</string>
     </property>
     <property name="buddy">
      <cstring>quternionZDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="quternionZDoubleSpinBox">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="quaternionWLabel">
     <property name="text">
      <string>w:</string>
     </property>
     <property name="buddy">
      <cstring>quternionWDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="quternionWDoubleSpinBox">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quternionWDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CoreQuaternionEditorWidget</receiver>
   <slot>updateValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quternionXDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CoreQuaternionEditorWidget</receiver>
   <slot>updateValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quternionYDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CoreQuaternionEditorWidget</receiver>
   <slot>updateValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quternionZDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CoreQuaternionEditorWidget</receiver>
   <slot>updateValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>valueChanged(osg::Quat)</signal>
  <slot>setValue(osg::Quat)</slot>
  <slot>updateValue()</slot>
 </slots>
</ui>
